src/features/DisplayControls/NursingTasks/styles/AdministeredMedicationList.scss

```diff
@@ -0,0 +1,25 @@
+.bx--structured-list {
+  margin: 0;
+}
+
+.time,
+.status {
+  position: relative;
+  padding-left: 8px;
+}
+
+.time {
+  top: -3px;
+}
+
+.status {
```

Comment:
these are very generic class names and might override with other similar class names... we have 2 options for that either give a unique classname to the top element of component and put all these classes under that hierarchy in scss or give each class unique names...better option is to use the first approach when we have multiple class names